### ISSUE TITLE
Audio feedback when moving or running into objects

### ISSUE THREAD

**Issue by emclaren** [mozilla/hubs#2811]
We've received feedback from two different groups involved with web accessibility that there should be some sort of audio feedback when you move & run into objects. 

To quote a survey response from the XR Access event: 

There is no feedback when you are moving. There should be a footstep or some kind of sound to say that you are moving and how fast.
There is no feedback when you run into something. There should be an "oof" sound or clang, telling you that direction is blocked.
